user/deployment/bluemixcloudfoundry.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Bluemix CloudFoundry Deployment
+layout: en
+permalink: /user/deployment/bluemixcloudfoundry/
+---
+
+You now have the amazing ability to deploy directly to [IBM Bluemic](https://console.ng.bluemix.net/) after a successful build on Travis CI.
+
+## Getting on the Edge
+
+Proper Bluemix CloudFoundry support is currently included only in the edge version of Travis.  See how to enable it via the `.travis.yml` below.
+
+## The Easy Way
+
+Go grab [the Travis gem from GitHub](https://github.com/travis-ci/travis.rb) and run this command:
+
```

Comment:
```sh
    travis setup bluemixcloudfoundry
    ```

Also add `sh` to the code block near the bottom on line 44

makes more sense here I think